src/UI/Art/SpriteArtist/UnitSpriteSet.java

```diff
@@ -115,6 +118,9 @@ public UnitSpriteSet(String unitType, Faction faction, ColorPalette coColors)
       }
     }
 
+    // Make a mask that we can draw with varying opacity to indicate buff effects.
+    buffMask = new Sprite(sprites[AnimState.TIRED.ordinal()]);
+    buffMask.convertToMask(new Color(255, 255, 255, 255));
```

Comment:
I would argue for `IDLE` over `TIRED`.

Reply:
potato potato. They have the same outline, yeah? I'm just converting it to a mask anyway.

Reply:
Technically, the user can provide a `TIRED` sprite, because of how we load things.
I don't really... *expect* that, but y'know.

Reply:
Huh. I guess they can. I don't think it used to be that way. Maybe we should just adjust `TIRED`'s `toString` to `''` so it agrees. I guess that would preclude fancy ridiculous tired animations for moved units. Which is definitely a priority feature we are likely to see in our lifetimes.

Sighfinewhatever.

Reply:
> Huh. I guess they can. I don't think it used to be that way.

True facts.
> Maybe we should just adjust `TIRED`'s `toString` to `''` so it agrees.

I'm not opposed to this.
> I guess that would preclude fancy ridiculous tired animations for moved units. Which is definitely a priority feature we are likely to see in our lifetimes.

I lol'd